engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-composite-content-views-list.html

```diff
@@ -67,7 +67,7 @@
           </div>
           <div>
             <a ui-sref="content-views.details.puppet-modules.list({contentViewId: contentViewVersion.content_view.id})" translate>
-              {{ contentViewVersion.puppet_modules.length || 0 }} Puppet Modules
+              {{ contentViewVersion.puppet_module_count || 0 }} Puppet Modules
```

Comment:
Won't this property be under the components ? e.g. `contentViewVersion.components.puppet_module_count`